presto-main/src/main/java/com/facebook/presto/operator/OperatorContext.java

```diff
@@ -264,6 +264,12 @@ public AggregatedMemoryContext aggregateUserMemoryContext()
         return new InternalAggregatedMemoryContext(operatorMemoryContext.aggregateUserMemoryContext(), memoryFuture, this::updatePeakMemoryReservations, false);
     }
 
+    // caller shouldn't close this context as it's managed by the OperatorContext
+    public AggregatedMemoryContext aggregateRevocableMemoryContext()
```

Comment:
This looks quite general, surprised not added before when adding aggregation spilling